### PR TITLE
[SPARK-28518][SQL][TEST] Refer to ChecksumFileSystem#isChecksumFile to fix StatisticsCollectionTestBase#getDataSize

### PR DESCRIPTION
**Pull request by wangyum**
## What changes were proposed in this pull request?

This PR fix [StatisticsCollectionTestBase.getDataSize](https://github.com/apache/spark/blob/8158d5e27fce8e4bc5877ed7bb4f7c3876007c13/sql/core/src/test/scala/org/apache/spark/sql/StatisticsCollectionTestBase.scala#L298-L304) refer to [ChecksumFileSystem.isChecksumFile](https://github.com/apache/hadoop/blob/release-2.7.4-RC0/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/ChecksumFileSystem.java#L93-L97).

More details: https://github.com/apache/spark/pull/25014#discussion_r307050435

## How was this patch tested?

unit tests

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
cc @gatorsmile

**Review by gatorsmile — COMMENTED** [review]
LGTM pending tests.

**Comment by gengliangwang** [comment]
retest this please.

File: sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala [hunk 1]
```diff
@@ -1453,4 +1453,35 @@ class StatisticsSuite extends StatisticsCollectionTestBase with TestHiveSingleto
       }
     }
   }
+
+  test("Fix StatisticsCollectionTestBase#getDataSize refer to ChecksumFileSystem#isChecksumFile") {
```
Comment (dongjoon-hyun) [inline sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala]: Please add `SPARK-28518` prefix to the test case name.

Comment (wangyum) [reply sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala]: OK. Done

Comment (dongjoon-hyun) [reply sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala]: Thanks!

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Merged to master.
Thank you, @wangyum , @gatorsmile , @gengliangwang !

**Pull request closed without merging**